docs/batch_inference_with_ts.md

```diff
@@ -106,7 +106,7 @@ $ curl -X POST "localhost:8081/models?url=resnet-152.mar&batch_size=8&max_batch_
 $ curl localhost:8081/models/resnet-152
 {
   "modelName": "resnet-152",
-  "modelUrl": "https://s3.amazonaws.com/model-server/model_archive_1.0/examples/resnet-152-batching/resnet-152.mar",
+  "modelUrl": "<local_OR_cloud_url>/resnet-152.mar",
```

Comment:
@gunandrose4u This needs to be a working url for the documentation example. We can consider adding .mar file to github

Reply:
+1 sounds good to me

Reply:
The link @gunandrose4u has removed is incorrect hence suggestions is to add links from this https://github.com/pytorch/serve/blob/master/docs/model_zoo.md